### PR TITLE
Added user bluetooth service name to device-info bt sample

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Instead of having a hard-coded `generic_access` bluetooth service, let's give user the ability to scan for a nearby device of his choice.

R: @jeffposnick

### CODE REVIEW

File: web-bluetooth/device-info.html [hunk 1]
```diff
@@ -9,22 +9,63 @@
 <p>This sample illustrates the use of the Web Bluetooth API to retrieve basic
 device information from a nearby Bluetooth Low Energy Device.</p>
 
-<button>Get Bluetooth Device Info</button>
+<form>
+  <label for="service">
+    Enter a Bluetooth Service name:
+    <input id="service" type="text" list="services" autofocus />
```
Comment (jeffposnick) [inline web-bluetooth/device-info.html]: You can get rid of the self-closing `/` at the end of the tag.

File: web-bluetooth/device-info.html [hunk 2]
```diff
@@ -9,22 +9,63 @@
...
+  <button>Get Bluetooth Device Info</button>
+</form>
 {% capture initial_output_content %}
 {% endcapture %}
 {% include output_helper.html initial_output_content=initial_output_content %}
 
 {% capture js %}
-function onButtonClick() {
+function onFormSubmit() {
+  var filterService = document.getElementById('service').value.trim();
```
Comment (jeffposnick) [inline web-bluetooth/device-info.html]: Why's the `.trim()` needed? I wouldn't assume that users would enter whitespace around their service name.

File: web-bluetooth/device-info.html (hunk 1 above)
Comment (beaufortfrancois) [inline web-bluetooth/device-info.html]: Done.

File: web-bluetooth/device-info.html (hunk 2 above)
Comment (beaufortfrancois) [inline web-bluetooth/device-info.html]: Removed.

**Comment by beaufortfrancois** [comment]
@jeffposnick Thanks for the review. 
I've addressed both nits.

**Comment by jeffposnick** [comment]
:+1:

**Pull request merged**